packages/Breadcrumbs/src/Breadcrumbs.js

```diff
@@ -11,14 +12,15 @@ import * as sc from "./Breadcrumbs.styles";
 function Breadcrumbs(props) {
   const { children, isDark, isAutoCollapsed, ...moreProps } = props;
   const I18n = useI18n();
-  const childrenCount = React.Children.count(children);
-  const shouldShowExpandButton = isAutoCollapsed && React.Children.count(children) > MAXIMUM_NUM_OF_LEVEL;
-  const hasOnlyOneChild = childrenCount === 1;
+
   const [isCollapsed, setIsCollapsed] = React.useState(shouldShowExpandButton);
+  const linkChildren = [extractChildren(children, ["Breadcrumbs.Link"])["Breadcrumbs.Link"]].flat();
```

Comment:
what's the difference between this and `extractChildren(children, ["Breadcrumbs.Link"])["Breadcrumbs.Link"]` ? For removing null values?

Reply:
if there's only one `<Breadcrumbs.Link>`, then `linkChildren` would be set to that component.  If there are multiple, then `linkChildren` would be an array.  

With my version here, it will always be an array, even if there's only one item.